tests/equilibria/test_physics.py

```diff
@@ -28,3 +32,30 @@ def test_psi_bd(self):
 
         assert round(abs(psib - 143), 0) == 0  # 142.66 ~ 143 V.s
         # CREATE DEMO 2015 test case
+
+
+class TestCoulombLogarithm:
+    @classmethod
+    def setup_class(cls):
+        df = pd.DataFrame(
+            columns=["Case", "n [1/m^3]", "T [eV]", "ln Lambda (Goldston)"]
```

Comment:
Possibly could use `https://github.com/hgrecco/pint-pandas` if you want units in pandas Dataframes